src/jsonrpc-machine-c-api.h

```diff
@@ -40,6 +40,8 @@ typedef enum cm_jsonrpc_cleanup_call {
 
 /// \brief Spawns a new remote machine server.
 /// \param address Address (in local host) to bind the new remote machine server.
+/// \param spawn_timeout_ms Maximum time in milliseconds to wait for the remote server to spawn before giving up
+/// and returning an error. Use -1 for no timeout.
 /// \param new_m Receives the pointer to the new JSONRPC remote machine object. Set to NULL on failure.
 /// \param bound_address_bound Receives the address that the remote machine server actually bound to,
```

Comment:
typo

```suggestion
/// \param bound_address Receives the address that the remote machine server actually bound to,
```

Reply:
Pushed a commit with a bunch of typo fixes in the C API.